php-frontend/src/main/java/org/sonar/plugins/php/api/tree/Tree.java

```diff
@@ -880,11 +885,6 @@
      */
     GOTO_STATEMENT(GotoStatementTree.class),
 
-    /**
-     * {@link YieldStatementTree}
-     */
-    YIELD_STATEMENT(YieldStatementTree.class),
```

Comment:
this needs to be documented in the ticket. This is a breaking change.

Reply:
updated https://jira.sonarsource.com/browse/SONARPHP-902